e2e/js_run_devserver/multirun_test.sh

```diff
@@ -14,107 +14,119 @@ _sedi () {
   sed "${sedi[@]}" "$@"
 }
 
-echo "TEST - $0: $1"
+echo "$$: TEST - $0: $1"
 
-./node_modules/.bin/ibazel run "$1" "$BZLMOD_FLAG" >/dev/null 2>&1 &
+./node_modules/.bin/ibazel run "$1" "$BZLMOD_FLAG" 2>&1 &
 ibazel_pid="$!"
 
 function _exit {
-  echo "Cleanup..."
+  echo "$$: Cleanup..."
   kill "$ibazel_pid"
+  wait "$ibazel_pid"
   git checkout src/index.html src/BUILD.bazel
   rm -f src/new.html
 }
 trap _exit EXIT
 
-echo "Waiting for $1 devserver to launch on 8080..."
+echo "$$: Waiting for $1 devserver to launch on 8080..."
 
+n=0
 while ! nc -z localhost 8080; do
```

Comment:
This was possibly an infinite loop before, and I did see CI go for 3+h stuck in it a few times.